client/src/components/Navbar/useStyles.ts

```diff
@@ -8,9 +8,23 @@ export const useStyles = makeStyles((theme: Theme) => ({
     background: 'white',
   },
   transparentNavbar: {
+    position: 'fixed',
+    zindex: 10000,
```

Comment:
```suggestion
    zIndex: 10000,
```
I think this is a typo here